internal/component/local/file_match/file.go

```diff
@@ -28,6 +28,7 @@ func init() {
 type Arguments struct {
 	PathTargets []discovery.Target `alloy:"path_targets,attr"`
 	SyncPeriod  time.Duration      `alloy:"sync_period,attr,optional"`
+	IgnoreOlder time.Duration      `alloy:"ignore_older,attr,optional"`
```

Comment:
@dehaansa about the name you mentioned that something along the lines of "drop_older" might fit better?

Reply:
I'm fine with both, I like the added consistency but I also find "ignore" a bit clearer than "drop" in this context.

Reply:
I also think `ignore_older_than` is more clear than `drop_older_than`.

`local.file_match` creates brand new data out of the files it discovers, so technically it isn't dropping anything. It's just ignoring older files.

Reply:
Should I make the change on the config name ? to `ignore_older_than` it seems more aligned in this context ?

Reply:
yes that's a good idea, thanks!

Reply:
@wildum I have made the changes. Please take a look